contract_price_revision/views/contract_line.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This is correct and mustn't be removed. Where you have to remove it is on .html files, not here.